doc/interop-test-descriptions.md

```diff
@@ -1011,16 +1011,49 @@ languages. Therefore they are not part of our interop matrix.
 #### rpc_soak
 
 The client performs many large_unary RPCs in sequence over the same channel.
-The number of RPCs is configured by the experimental flag, `soak_iterations`.
+
+This test is configurable via a few different command line flags:
```

Comment:
Are these flags mandatory? Is it okay to have a default if it isn't provided? Can we suggest a common default?

Reply:
I think these flags really need to be available in the binary in order for it to be useful.

The actual settings of the flags will differ a lot depending on the exact environment that the test is ran in. For example, we currently run C++ versions of this test in primarily two contexts:
a) Client is in a different GCP region from the server
b) Client is in the same GCP zone as the server

These tests require significantly different values for all of these flags, mainly since round trip time and network reliability differs a lot between the two cases.

I'm not sure it makes sense to tailor this test settings to a particular context, since it's not clear what that context should be. And further, it may cause confusion if the test fails when ran outside of that context.